git/refs/log.py

```diff
@@ -71,7 +71,7 @@ def actor(self):
         return self[2]
 
     @property
-    def time(self):
+    def time_(self):
```

Comment:
Is changing the `time` property name truly required? It would break people using it. If possible, I would prefer to keep it as is and silence deep source in other ways.